src/display.rs

```diff
@@ -236,7 +236,6 @@ impl Display {
     /// Currently this only works on X11 with XComposite and XDamage extensions available.
     ///
     /// Deprecated since GDK 3.16
-    #[cfg(not(feature = "gdk_3_16"))]
```

Comment:
Yes, that's what I said. :-) Then everything's good. I merge (and update crates ?).